### DISCUSSION TITLE
Are there two icon versions for Arcticons You?

### DISCUSSION THREAD

**Discussion by Crazy-Unicooorn** [Arcticons-Team/Arcticons#1677] [Q&A] [answered]
Hey there!
Basically, I have an issue and I am having a hard time knowing if it is related to my icon pack (Arcticons You) or my launcher (Kvaesitso).

We are already discussing it in Kvaesitso's repo: [https://github.com/MM2-0/Kvaesitso/issues/519#issuecomment-1696441506](https://github.com/MM2-0/Kvaesitso/issues/519#issuecomment-1696441506)

I will start with the question and then explain my issue to provide more context:

### Is it Arcticons You that provides the following 2 versions of the icons?
(and if so, did anything change related to that with the latest update?)

One with a dark colored background and light colored lines:
![image](https://github.com/Donnnno/Arcticons/assets/30509883/d37bcd21-f545-437a-8213-d14befa53c3f)

An other with a grey dark background and subtle colored lines:
![image](https://github.com/Donnnno/Arcticons/assets/30509883/1b51b0ca-2b2b-4cf6-b0b8-054c30c9c0c4)

### The issue

On the launcher mentioned above, there is a toggle to switch between these 2 versions, and it stopped working for Arcticons You a couple days ago (but still works for Arcticons Dark and Light packs). Now I can only have the icons from the first screenshot, pressing the toggle won't do anything.

I doubt this is related to the icon pack, but any clarification is appreciated!

**Comment by Donnnno** [comment, marked as answer]
Replied in the original issue!

But no, there's only one version :+)
